portfolio/src/main/webapp/galleryStyle.css

```diff
@@ -0,0 +1,56 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
This changes `box-sizing` for *every* element in the document, is that what you want?

It might not break anything, but changing defaults like this can be error-prone in more complex applications (e.g. if you're including some library/widget with its own CSS that assumes default box-sizing).

Reply:
Got it! I was testing some things with formatting and included that, but I tried testing it again with this part removed, and everything still seems intact, so I'm going to leave the box-sizing part out.